### ISSUE TITLE
SVGKitSwift SPM build issue (when archiving only)

### ISSUE THREAD

**Issue by atb1983thrive** [SVGKit/SVGKit#724]
[X] I have read and understood the CONTRIBUTING guide
[X] I have reproduced the problem while running in Xcode, and read the Console log, and acted upon any relevant error or warning messages
[X] I have reproduced the problem while running in Xcode's Debugger, and used breakpoints to examine the actual values of variables vs the expected values
 I am not using CocoaPods, or: I am using CocoaPods, but have verified my podspec points to SVGKit v2.x
Issue Description and Steps

When adding SVGKit as an SPM dependence

I'm able to use SVGKit just fine. the issue is when archiving

I'm using Xcode 12.5.1 and SwiftUI
I tried using a fresh test project and always got this issue:

<img width="379" alt="Screenshot 2021-07-15 at 12 32 54" src="https://user-images.githubusercontent.com/77971847/125781583-bf18d49a-86bd-4594-aa6b-c831407f9679.png">

Could you please help me? thank you.

**Comment by adamgit** [comment]
I think this project really needs some more people who are heavily using
swift to start contributing code / doing testing. The core was built
originally in ObjC with bits of C (!), and I joined to heavily upgrade it
more on the ObjC side. Now it needs some people to more heavily upgrde the
Swift parts. Sorry I can't help more right now.

**Comment by bfolkens** [comment]
I think there's a fundamental issue here, whereby having the SVGKitSwift dependency in the package seems to indicate to the compiler that it must be built.  The only way I was able to get this to archive was to change the package spec to `.v13` - I'm also on 12.5.1

This obviously bumps the minimum version a lot higher, when it may not be necessary except for that one product (SVGKitSwift).

I should note that I also tried wrapping the entire file in `#if canImport(SwiftUI)`.

Some thoughts:

- Maybe this is just an issue in latest Xcode?  I haven't tested it on others.
- Change SVGKitSwift to SVGKitSwiftUI to avoid confusion (the name tripped me up on import, though made more sense after looking at Package.swift and the sources)
- Perhaps move SVGKitSwiftUI to it's own Package to avoid issues with backward compatibility if `.v9` is necessary.

I'll file a PR for convenience.  Though it's quick fix, I'm not sure if it's the most optimal solution, depending on what route @adamgit wants to take with this...
